llvm/test/Transforms/CorrelatedValuePropagation/min-max.ll

```diff
@@ -1,5 +1,6 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
-; RUN: opt < %s -passes=correlated-propagation -S | FileCheck %s
+; RUN: opt < %s -passes=correlated-propagation -S | FileCheck %s --check-prefixes=CHECK,CHECK-CANONICALIZE-ON
+; RUN: opt < %s -passes=correlated-propagation -canonicalize-icmp-predicates-to-unsigned=false -S | FileCheck %s --check-prefixes=CHECK,CHECK-CANONICALIZE-OFF
```

Comment:
Please remove the `-canonicalize-icmp-predicates-to-unsigned=false` check lines, we don't care about this. Feel free to remove the option in a followup.